backend/src/controllers/dish.ts

```diff
@@ -1,3 +1,41 @@
 import { Request, Response } from 'express'
 
-export const getDishes = async (req: Request, res: Response) => {}
+import { db } from '../services/firebase'
+import { Dish, Transaction } from '../utils/types'
+import { mapDishesToLatestTransaction, mapToDishVM } from '../utils/service/dish'
+
+export const getDishes = async (req: Request, res: Response) => {
+    // get dishes from firebase
+    let dishesQuerySnapshot = await db.collection('dishes').get()
```

Comment:
can you add try-and-catch statements for error handling and return 4xx or 5xx errors with json message

Reply:
you can use:
```
req.log.info({message: "something"});
req.log.error({error: e});
```